go/engine/device_wrap.go

```diff
@@ -122,8 +122,8 @@ func (e *DeviceWrap) refreshMe(m libkb.MetaContext) (err error) {
 	return err
 }
 
-func (e *DeviceWrap) setActiveDevice(m libkb.MetaContext) (err error) {
-	defer m.CTrace("DeviceWrap#setActiveDevice", func() error { return err })()
+func (e *DeviceWrap) SetActiveDevice(m libkb.MetaContext) (err error) {
```

Comment:
can you rename to `CommitNewUserConfigAndActiveDevice`? a comment here about when/why this method should be called separately from `Run` would be good too